src/pandora/service.ts

```diff
@@ -592,7 +598,8 @@ export class PandoraService {
         perEpoch: selectedCosts.perEpoch,
         perDay: selectedCosts.perDay,
         perMonth: selectedCosts.perMonth
-      }
+      },
+      depositAmountNeeded
```

Comment:
```suggestion
      depositAmountNeeded: lockupNeeded
```

how about we either name it `depositAmountNeeded` where it's first defined, or just use a rename when setting the property